dev/com.ibm.websphere.appserver.features/visibility/public/persistentExecutor-2.0/resources/l10n/com.ibm.websphere.appserver.persistentExecutor-2.0.properties

```diff
@@ -0,0 +1,18 @@
+###############################################################################
+# Copyright (c) 2017,2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables the creation of persistent scheduled executors that allow applications \
```

Comment:
Suggested edit:

This feature enables the creation of persistent scheduled executors that allow applications \
 to schedule persistent tasks that run with _a_ thread context that is managed by the application server.

Reply:
add article "a" before "thread context"

Reply:
This description was copied from the corresponding v1.0 feature, so I'm fixing it as you suggest in both places. Please see https://github.com/OpenLiberty/open-liberty/pull/13732/commits/d58aee9651a916c90c362bf4e9f98d4a2d03017f for the change